dev_envs/mac-env-setup.md

```diff
@@ -249,14 +249,21 @@ and paste the contents of your clipboard
 There are some additional setup steps to access the
 [COOL (Cloud-Optimized Operations Lab)](https://github.com/cisagov/cool-system/wiki/):
 
-1. [Install MIT Kerberos](https://github.com/cisagov/cool-system/wiki/Installing-MIT-Kerberos)
+**Note:**
+These steps are only necessary if you want to manually configure these vs
```

Comment:
Extremely minor suggestion.  "vs" is an abbreviation for "versus", and in American English abbreviations like this always end in a period.  (British English _does not_ use a period if the last letter of the abbreviation is the last letter of the word, which is also why they write "Mr" instead of "Mr.".) 
```suggestion
These steps are only necessary if you want to manually configure these vs.
```